build/bin/release

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+if [ "$#" -lt "1" ]; then
+    echo "must specify chart directory"
+    exit 1
+fi
+
+set -x
+
+chart=$1
+
+rm -r build/release
+
+owner=${GITHUB_USER:-onosproject}
+repo=${GITHUB_REPO:-onos-helm-charts}
+remote=${GIT_REMOTE:-origin}
+token=${GITHUB_TOKEN}
+branch=$(git branch --show-current)
```

Comment:
If this will be run by humans, should check that all changes have been committed, so you don't accidentally package a manually modified chart.